tmt/steps/__init__.py

```diff
@@ -68,8 +68,25 @@
 
 
 DEFAULT_ALLOWED_HOW_PATTERN: Pattern[str] = re.compile(r'.*')
-DEFAULT_PLUGIN_METHOD_ORDER: int = 50
 
+#
+# Following are default and predefined order values for various special phases
+# recognized by tmt. When adding new special phase, add its order below, and
+# do not forget to update the list in /spec/core/order. Please, keep the name
+# prefix, so it's easy to find all `PHASE_ORDER_*` constants for documentation.
+#
+
+#: The default order of any object.
+# TODO: this is a duplication of tmt.base.DEFAULT_ORDER. Unfortunately, tmt.base
+# imports tmt.steps, not the other way around.
```

Comment:
What about dropping `DEFAULT_ORDER = 50` from `base.py` instead?

Reply:
It feels correctly placed there, it's the value of `order` after all, which applies to all objects, not just to steps.

Reply:
Ah, right, it's not just steps.